includes/Shopping/Shopify_Query.php

```diff
@@ -231,9 +231,35 @@ protected function fetch_remote_products( string $search_term ) {
 		 */
 		$result = json_decode( $body, true );
 
-		// TODO(#11268): Error handling.
 		if ( isset( $result['errors'] ) ) {
-			return new WP_Error( 'rest_unknown', __( 'Error fetching products', 'web-stories' ), [ 'status' => 404 ] );
+			$wp_error = new WP_Error();
+
+			try {
```

Comment:
Why does this need to be in a try / catch?

Reply:
Ya thinking I can just remove at this point --- 

Any thoughts on looping over the errors vs just grabbing the first one ?    I think just grabbing the first should be enough.

Reply:
Removed the try catch now